caleb-lab/data/lyric.txt

```diff
@@ -0,0 +1 @@
+I won't deny it I'm a straight rida,
```

Comment:
No points lost since this was a lab back from Week 1, but please get in the habit of putting comments on your own branch for your own reference, and submitting a clean branch to Canvas.